testfiles/5ZoneAirCooledWithSpacesHVAC.idf

```diff
@@ -3193,7 +3193,32 @@ ZoneAirHeatBalanceAlgorithm,
     4.6,                     !- Economizer Minimum Limit Dry-Bulb Temperature {C}
     NoLockout,               !- Lockout Type
     FixedMinimum,            !- Minimum Limit Type
-    Min OA Sched;            !- Minimum Outdoor Air Schedule Name
+    ,                        !- Minimum Outdoor Air Schedule Name
+    ,                        !- Minimum Fraction of Outdoor Air Schedule Name
+    ,                        !- Maximum Fraction of Outdoor Air Schedule Name
+    Controller Mechanical Ventilation 1;  !- Mechanical Ventilation Controller Name
+
+  Controller:MechanicalVentilation,
+    Controller Mechanical Ventilation 1,  !- Name
```

Comment:
Add a Controller:MechanicalVentilation to testfiles/5ZoneAirCooledWithSpacesHVAC.idf